backend/validators/task.js

```diff
@@ -2,6 +2,6 @@ const { body } = require('express-validator');
 
 const store = [body('name').trim().notEmpty().withMessage('Task name is required')];
 
-const update = [...store, body('completed').trim().notEmpty().withMessage('Task completed flag is required')];
+const update = [...store, body('completed').notEmpty().withMessage('Task completed flag is required')];
```

Comment:
I think there we can add also `isBoolean`

Reply:
done